scripts/jenkins-yoctobuild-build.sh

```diff
@@ -824,7 +839,7 @@ build_and_test_client() {
 
             local pytest_args=
             if ! ( is_poky_branch morty || is_poky_branch pyro || is_poky_branch rocko || is_poky_branch sumo ); then
-                pytest_args="--no-pull"
+                pytest_args="--no-pull --commercial-tests"
```

Comment:
This will break thud, but if this works, I plan to cherry-pick it there immediately.

Reply:
Ok. What about sumo/rocko?

Reply:
See the `if` condition. The branches are already excluded.